Update mattercon2021.md (#725)
mattermost/mattermost-handbook: contributors/mattercon/mattercon2021.md

MatterCon 2021 will be Mattermost's first VR conference. Using Oculus Quest 2 devices, we'll be meeting up virtually from June 22 - 24, 2021. [Join the MatterCon2021 channel on Community](https://community.mattermost.com/core/channels/mattercon-2021). Use the hashtag #mattercon2021.

## Community engagement information

We'll have multiple opportunities for staff members to engage with the community on June 24th, our official Community Day.

1. Submit AMA questions for our top contributor panel [here](https://forms.gle/FGpmm7DhvrciQqgr7).
2. Sign up to be a Community Buddy [here](https://forms.gle/FGpmm7DhvrciQqgr7).

### What's expected of a Community Buddy?

> A Community Buddy is an internal staff member who is paired up with a member of our community. We'll default to pairing people up via timezones to be most convenient for them. However if there's a specific community member you'd like to work with, please indicate that on the form above.

Reviewer: ```suggestion
A Community Buddy is an internal staff member who's paired up with a member of our community. We'll default to pairing people up based on time zones to be most convenient for them. However if there's a specific community member you'd like to work with, please indicate that on the form above.
```
